habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -319,6 +319,13 @@ class AuxLossConfig(HabitatBaselinesBaseConfig):
     pass
 
 
+@dataclass
+class BdpDiscrimConfig(AuxLossConfig):
+    loss_scale: float = 0.1
+    hidden_size: int = 128
+    behavior_latent_dim: int = -1
+
+
 @dataclass
 class CPCALossConfig(AuxLossConfig):
```

Comment:
Isn't this name misleading? If I understand the code, you are currently using a classification loss, not CPC.

Reply:
What do you mean by CPC? I shouldn't have added anything related to CPC?

Reply:
Sorry, I thought the CPCALoss had been added.